package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@quintype/backend",
-  "version": "1.3.0",
+  "version": "1.3.0-upgrade-package.1",
```

Comment:
remove this while merging the pr

Reply:
it gets automatically updated

Reply:
hey, it looks like a beta version, can you verify this.

Reply:
no can you take a look into standard-version-release file.